content/en/containers/guide/container-discovery-management.md

```diff
@@ -152,51 +222,76 @@ spec:
+In your Helm chart, set `datadog.excludePauseContainer` to `true` or `false`.
 
 ##### Example
 
 ```yaml
 datadog:
   containerExclude: "image:<IMAGE_NAME_1> image:<IMAGE_NAME_2>"
   containerInclude: "image:<IMAGE_NAME_3> image:<IMAGE_NAME_4>"
+  excludePauseContainer: false
 ```
 
 {{% /tab %}}
 {{% tab "Containerized Agent" %}}
 
 In environments where you are not using Helm or the Operator, the following environment variables can be passed to the Agent container at startup.
 
-##### Example Docker
-```shell
-docker run -e DD_CONTAINER_EXCLUDE=image:<IMAGE_NAME> ...
-```
-
-##### Example ECS
-```json
-"environment": [
-  {
-    "name": "DD_CONTAINER_EXCLUDE",
-    "value": "image:<IMAGE_NAME>"
-  },
-  ...
-]
-```
-
+Set `DD_EXCLUDE_PAUSE_CONTAINER` to `false`.
 {{% /tab %}}
 {{< /tabs >}}
 
-#### Pause containers
+## Advanced CEL exclusion
 
-The Datadog Agent excludes Kubernetes and OpenShift pause containers by default. This prevents their metric collection and counting as billable containers. They are still counted in the container count metrics such as `kubernetes.containers.running` and `docker.containers.running`.
+In **Agent v7.73+**, you can use the `cel_workload_exclude` configuration option to filter containers from Autodiscovery. This feature allows you to define [Common Expression Langauge][3] rules to target containers to be excluded from telemetry collection.
 
-To disable this behavior and include monitoring the pause containers:
+The following attributes representing the container object are available for your filtering rules:
+
+| Attribute                   | Description                                                             |
+|-----------------------------|-------------------------------------------------------------------------|
+| `container.name`            | The name of the container.                                              |
+| `container.image.reference` | The full reference of the container image (registry, repo, tag/digest). |
+| `container.pod.name`        | The name of the pod running the container.                              |
+| `container.pod.namespace`   | The Kubernetes namespace of the pod.                                    |
+| `container.pod.annotations` | The annotations applied to the pod (key-value map).                     |
+
+### Configuration structure
+
+The `cel_workload_exclude` configuration option is structured as a list of rule sets joined by logical **ORs**. Each rule set defines the `products` to exclude and the corresponding CEL `rules` to match against containers.
+
+The `products` field accepts `metrics`, `logs`, and `global` (exclude container from all listed products).
+
+<div class="alert alert-danger">
+If the configuration contains structural errors or CEL syntax issues, the Agent exits with an error to prevent collecting unintended telemetry which could impact billing.
```

Comment:
```suggestion
If the configuration contains structural errors or CEL syntax issues, the Agent exits with an error to prevent collecting unintended telemetry that could impact billing.
```